cpp/src/gandiva/gdv_function_stubs.cc

```diff
@@ -427,7 +427,8 @@ CAST_VARLEN_TYPE_FROM_NUMERIC(VARBINARY)
 #undef GDV_FN_CAST_VARCHAR_REAL
 
 GANDIVA_EXPORT
-int32_t gdv_fn_utf8_char_length(char c) {
+FORCE_INLINE
```

Comment:
I think add a separate macro in this file
```
#if defined(_MSC_VER)
#define GDV_FORCE_INLINE __forceinline
#else
#define GDV_FORCE_INLINE inline __attribute__((always_inline))
#endif
```

Reply:
I applied that change, lets see if the tests will work